tests/test_hadoop_conf.py

```diff
@@ -103,7 +104,7 @@ def test_get_resource_endpoint(self):
 
                 endpoint = hadoop_conf.get_resource_manager_endpoint()
 
-                self.assertEqual('example.com:8022', endpoint)
+                self.assertEqual('http://example.com:8022', endpoint)
                 parse_mock.assert_called_with(hadoop_conf_path + 'yarn-site.xml',
```

Comment:
why are these tests not failing without these changes?

Reply:
I think the test changed because result value now will include schema 
https://github.com/toidi/hadoop-yarn-api-python-client/blob/f88ac6966eff575b18563ef8829967546cf3cc3f/yarn_api_client/hadoop_conf.py#L43